packages/terra-collapsible-menu-view/src/CollapsibleMenuViewHyperlink.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Hyperlink from 'terra-hyperlink';
+import Menu from './_CollapsibleMenu';
+import styles from './CollapsibleMenuView.module.scss';
```

Comment:
You are pulling styles from a different component module into your component.

Reply:
With the way that collapsible menu view was, all the scss was in one file. Unfortunately I don't have the time to split out all the scss into separate files for time sake